roles/calico/defaults/main.yaml

```diff
@@ -4,7 +4,11 @@ etcd_endpoints: "{{ hostvars[groups.oo_first_master.0].openshift.master.etcd_url
 
 cni_conf_dir: "/etc/cni/net.d/"
 cni_bin_dir: "/opt/cni/bin/"
+cni_url: "https://github.com/containernetworking/cni/releases/download/v0.4.0/cni-amd64-v0.4.0.tgz"
 
 calico_etcd_ca_cert_file: "/etc/origin/calico/calico.etcd-ca.crt"
 calico_etcd_cert_file: "/etc/origin/calico/calico.etcd-client.crt"
 calico_etcd_key_file: "/etc/origin/calico/calico.etcd-client.key"
+
```

Comment:
Maybe 

```yaml
calico_url_prefix: "https://github.com/projectcalico/cni-plugin/releases/download/v1.5.5"
calico_url_cni: "{{ calico_url_prefix }}/calico"
calico_url_ipam: "{{ calico_url_prefix }}/calico-ipam"
```

to be implicitly sure both `calico` and `calico-ipam` are of the same version.